bridge/src/lib.rs

```diff
@@ -615,7 +611,8 @@ impl<T: Config> Pallet<T> {
 		let fee_integer = T::NativeTokenDecimals::get() * (integer as u128);
 		let fee_fraction =
 			T::NativeTokenDecimals::get() * (fraction.deconstruct() as u128) / 1_000_000_000;
-		let ft_fee: BalanceOf<T> = (fee_integer + fee_fraction).checked_into().unwrap();
+		let ft_fee: BalanceOf<T> =
+			(fee_integer + fee_fraction).checked_into().expect("Fee overflow");
```

Comment:
Should we avoid crash here? @anonymousGiga

Reply:
> Should we avoid crash here? @anonymousGiga
Yes,

Maybe function ```calculate_and_update_fee``` should return an error, the correct code  like this:
```
fn calculate_and_update_fee(near_price: u64, native_token_price: u64) -> DispatchResult {
      ...
      let ft_fee: BalanceOf<T> = (fee_integer + fee_fraction).checked_into()..ok_or(Error::<T>::InvalidFeeMakeOverflow)?;
      ...
}
```